src/App.unit.test.tsx

```diff
@@ -0,0 +1,25 @@
+type Movie = {
```

Comment:
You should be able to import the type from the file `types/movie` instead rewrite this type